### DISCUSSION TITLE
Steam Next Fest October 2022 - Demo AppIds

### DISCUSSION THREAD

**Discussion by Poetic-Abortionist** [TCNOco/TcNo-Mass-Steam-Idler#8] [General]
Hey all!

Here's a ~~[list](https://pastebin.com/kUarHGVU) off all (at that time) 723 appIDs~~ **Fixed** the previous list [they're now 711 titles](https://pastebin.com/3pyu0LE7) and they seem to work. I extracted from the Steam website. I can't test it because the idler doesn't work on my PC hope it works on your end.

**Hello again there isn't a badge for idling games but there's one for looking at your discovery queue. I jumped the gun and assumed that the badge was for playing demos but unfortunately that isn't the case.**

**Comment by verdammnis** [comment]
10 activations failed. Most likely wrong appIds

**Reply by Poetic-Abortionist** [reply]
Sorry about that [here's](https://pastebin.com/3pyu0LE7) another list containing the correct IDs of 711 demos I tested them in a VM and they seem to work.

**Reply by verdammnis** [reply]
> Sorry about that [here's](https://pastebin.com/3pyu0LE7) another list containing the correct IDs of 711 demos I tested them in a VM and they seem to work.

No problem it really works. Thank you very much :)
